src/status_im/ui/screens/wallet/send/sheets.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im.ui.screens.wallet.send.sheets
+  (:require-macros [status-im.utils.views :refer [defview letsubs] :as views])
+  (:require [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.list-item.views :as list-item]
+            [status-im.ui.components.chat-icon.screen :as chat-icon]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui.screens.wallet.accounts.views :as wallet.accounts]
+            [status-im.utils.utils :as utils.utils]))
+
+(views/defview assets [address]
+  (views/letsubs [{:keys [tokens]} [:wallet/visible-assets-with-values address]
+                  currency [:wallet/currency]]
+    [list/flat-list
+     {:data      tokens
+      :key-fn    (comp str :symbol)
+      :render-fn (wallet.accounts/render-asset
+                  (:code currency)
+                  #(do
```

Comment:
is code that got moved? could you create an event to avoid the double dispatch?

Reply:
will fix to one event